### PR TITLE
Email confirmation

### PR DESCRIPTION
**Pull request by VolodyaAll**
Send email to specify email addres with link to login page with confirmation message

### CODE REVIEW

**Comment by stahor** [comment]
Используй sendgrid

**Comment by showthreadx** [comment]
Нужно использовать figaro вместо dotenv

File: app/controllers/checkins_controller.rb [hunk 1]
```diff
@@ -7,7 +7,6 @@ def create
   end
 
   private
-
     def checkin_params
```
Comment (valkovik15) [inline app/controllers/checkins_controller.rb]: Why remove empty line?

File: app/controllers/trips_controller.rb [hunk 2]
```diff
@@ -18,7 +18,6 @@ def show
   end
 
   private
-
     def trip_params
```
Comment (valkovik15) [inline app/controllers/trips_controller.rb]: And here

File: app/controllers/checkins_controller.rb (hunk 1 above)
Comment (valkovik15) [reply app/controllers/checkins_controller.rb]: Or was it rubocop -a? Ok, then

Comment (VolodyaAll) [reply app/controllers/checkins_controller.rb]: Yes, it was rubocop -a )

File: app/controllers/trips_controller.rb (hunk 2 above)
Comment (VolodyaAll) [reply app/controllers/trips_controller.rb]: And here

**Pull request merged**